### PR TITLE
Microsoft.AnalysisServices-Servers-Max param file

### PR DESCRIPTION
**Pull request by SaumilkumarShah**
# Change

Microsoft.AnalysisServices-Servers-Max param file

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Comment by SaumilkumarShah** [comment]
Validated from code to deploy resource

image

**Comment by MrMCake** [comment]
@SaumilkumarShah could you please attach the pipeline/badge to the PR? I trust you based on your screenshot that it worked, but would like to review the logs regardless :) We can (unfortantely) not run tests on our end on a fork.

File: arm/Microsoft.AnalysisServices/servers/.parameters/max.parameters.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            "value": [
+                {
+                    "roleDefinitionIdOrName": "Reader",
+                    "principalIds": [
+                        "<<deploymentSpId>>"
+                    ]
+                }
+            ]
+        },
+        "cuaId": {
```
Comment (MrMCake) [inline arm/Microsoft.AnalysisServices/servers/.parameters/max.parameters.json:51]: Please remove

Comment (SaumilkumarShah) [reply arm/Microsoft.AnalysisServices/servers/.parameters/max.parameters.json]: Removed

**Review by MrMCake — APPROVED** [review]

**Pull request merged**